.pre-commit-config.yaml

```
@@ -58,7 +58,7 @@ repos:
           - -d
           - '{extends: relaxed, rules: {line-length: {max: 90}}}'
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.0.284
+    rev: v0.0.285
```

ruff 0.0.285 is broken
```suggestion
rev: v0.0.285
rev: v0.0.284
```